core/contractsdk/java/src/main/java/com/baidu/xuper/Driver.java

```diff
@@ -0,0 +1,41 @@
+package com.baidu.xuper;
+
+import java.net.InetSocketAddress;
+
+import io.grpc.Server;
+import io.grpc.netty.shaded.io.grpc.netty.NettyServerBuilder;
+
+/**
+ * Driver
+ */
+public class Driver {
+    final private static String XCHAIN_CHAIN_ADDR = "XCHAIN_CHAIN_ADDR";
```

Comment:
Any comments for those hard-code environmen?

Reply:
done